Rakefile

```diff
@@ -127,14 +129,33 @@ end
 
 ##################################### Dev ######################################
 namespace :dev do
+
+  # docker-compose command with root dev args
+  dc = 'docker-compose -f docker-compose-dev.yml'
+
   desc 'Launches the dev environment with docker-compose'
   task :up do
-    system 'docker-compose -f docker-compose-dev.yml up --build -d'
+    system "#{dc} up --build -d"
   end
 
-  desc 'Brings down the dev environment'
+  desc 'Stop and remove the dev environment (containers, networks, volumes, etc)'
   task :down do
-    system 'docker-compose -f docker-compose-dev.yml down'
+    system "#{dc} down"
+  end
+
+  desc 'Start existing services previously stopped with dev:stop'
+  task :start do
+    system "#{dc} start"
+  end
+
+  desc 'Stop running services without removing them'
+  task :stop do
+    system "#{dc} stop"
+  end
+
+  desc 'Force a complete rebuild of all containers without using cached layers'
+  task :rebuild do
+    system "#{dc} build --no-cache --progress tty"
```

Comment:
Just curious, what does the `tty` here do?

Reply:
From `docker-compose build -h`:

```sh
   --progress string       Set type of progress output (auto, plain, tty).
                            EXPERIMENTAL flag for native builder.
                            To enable, run with COMPOSE_DOCKER_CLI_BUILD=1)
```